packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -1,2 +1,8 @@
+<h2 data-translate="manager_hub_dashboard_overview"></h2>
+<hub-payment-status
+    data-services="$ctrl.billingServices.data"
+    data-total-count="$ctrl.billingServices.count"
```

Comment:
Isn't it your PR ? 🤔

Reply:
@ganeshkumar1989 you're right, I'm currently checking because it seems strange to be able to display any kind of services but have a filter on the count

Reply:
@FredericEspiau , yes it was my PR and I had a different code :)
Thanks for the update @marie-j